pkg/tiller/kind_sorter.go

```diff
@@ -123,5 +123,9 @@ func (k *kindSorter) Less(i, j int) bool {
 	if !ok {
 		return true
 	}
+	if first == second {
+		// same kind so sub sort alphanumeric
+		return a.name < b.name
+	}
```

Comment:
Summarizing your comment:
2 or more unknown but different types with same name

Im good with catching this in 2.6.1 as to not slow down 2.6.0. @seh do you think thats acceptable?